omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+    delete ``OriginalFile`` objects that are in the script repository,
+    which is where official scripts reside
+
+``ModifyGroup``
+    make changes to groups, such as their name or permissions level
+
+``ModifyGroupMembership``
+    make changes to who is a member or owner of which group
+
+``ModifyUser``
+    make changes to users, such as their name or institution
+
+``Sudo``
+    become another user
+
+``ReadSession``
+    read other users' session UUIDs
+
+
+Bundling restrictions
+---------------------
+
+It often makes sense to apply all but a certain bundle of restrictions
+to a user. For example, users working with other users' data may benefit
+from being able to go beyond their normal group restrictions only for
+the operations of ``WriteOwned``, ``WriteManagedRepo``,
+``WriteScriptRepo``, ``Chgrp``. A facility manager importing on behalf
+of other users may appropriately be given all but the ``Sudo``
+restriction. A human resources representative may be given all but the
+``ModifyGroup``, ``ModifyGroupMembership``, ``ModifyUser`` restrictions.
+Much depends on how personnel roles are handled in each specific
+institution.
+
+.. warning::
+
+    The ``ReadSession`` restriction should be applied to *all*
+    restricted administrators. Without that restriction members of the
+    ``system`` group can read other users' session UUIDs and join those
+    sessions as the user. In contrast, while an administrator without
+    the ``Sudo`` restriction can become other users, the security system
+    prevents their using sudo to elevate their own administrative
+    powers. An administrator cannot sudo to become ``root`` to escape
+    their restrictions. :javadoc:`A security filter
+    <ome/security/basic/LightAdminPrivilegesSecurityFilter.html>`
+    assists in enforcing ``ReadSession``.
+
+
+Working with restrictions
+-------------------------
+
+Restricted administrators
+"""""""""""""""""""""""""
+
+The :javadoc:`admin service <slice2html/omero/api/IAdmin.html>` offers
+new operations for managing restrictions on administrators.
+``createLightSystemUser`` creates a new restricted administrator.
+``getAdminPrivileges`` and ``setAdminPrivileges`` manage the
+restrictions on an existing administrator.
+
+Using ``setAdminPrivileges`` to set an empty list of privileges fills
+that user's ``Experimenter.config`` property with a ``false`` value for
+every ``AdminPrivilege`` name. That user does not thus become like a
+normal user: they retain all administrative powers not explicitly
+restricted, such as being able to read all users' images. A normal
+user's restrictions can be set: they simply have no effect if the user
+is not an administrator as there is then nothing to restrict.
+
+``getCurrentAdminPrivileges`` is useful for OMERO clients to find how
+the currently logged-in administrator is restricted.
+``getAdminsWithPrivileges`` identifies the administrators who are
+sufficiently unrestricted in a given way so that users know which
```

Comment:
who are ``users`` here ? The same users as mentioned above ? General real life people ? ``Experimenters`` in OMERO ? Would actually prefer if the part ``... so that...`` etc. would get deleted.